scripts/ChatterboxGetSpeakerData/ChatterboxGetSpeakerData.gml

```diff
@@ -0,0 +1,25 @@
+/// Returns the value between special characters 
+/// (CHATTERBOX_SPEAKERDATA_TAG_...) after the speaker and before the speech.
+///
+/// @param chatterbox
+/// @param contentIndex
+/// @param [default]
+
+function ChatterboxGetSpeakerData(_chatterbox, _index, _default = undefined)
+{
+    var _str = ChatterboxGetContent(_chatterbox, _index);
+    if (_str != undefined)
+    {
+        var _colon = string_pos(":", _str);
```

Comment:
Please make the colon presence check explicit (i.e. not `_char_r < _colon`) as this is clearer. Less clever, but clearer.

Reply:
Could you explain further? I'm not sure what you mean with explicit. I changed it to `if (_tag_l != 0 and _tag_r < _colon_pos)`, but I'm not sure if that's what you meant